Layout w HTML i CSS/product_page/design/solution.md

```diff
@@ -0,0 +1,21 @@
+Części:
+1. nawigacja (pod nawigacją dolny border)
+ po lewej
+ - logo
+ - 5 buttonów nawigacji
+ 
+ po prawej
+ -koszyk i zdjęcie avatar
+
+2. Oferta
+2.1 Zdjęcia
+    -Div - zdjęcie główne - div z tłem ze zdjęcia
+    -Div- 4 zdjęcia miniatur div z tłem ze zdjęcia
+
+2.2 Opis po prawej stronie zdjęć
+- header
+- główny header
+- Opis - paragraf
+-cena - header obk ceny procent przeceny
+-stara cena
+- Przciski- Div - flexbox/postion z 3 przyciskami (- + add to cart)
```

Comment:
Pewnie przyjdzie to wraz z praktyką, niemniej i tu zaznaczę, że zwykle opis zadań nie zawiera szczegółów implementacyjnych - na ogół zadanie raczej nie powinno narzucać inżynierowi sposobu wykonania, niemniej na ten moment bardzo fajnie rozpisane i na pewno przydatne. Daję okejkę 👍